bitbots_head_mover/config/head_config.yml

```diff
@@ -0,0 +1,212 @@
+move_head:
+  # Max values for the head position
+  max_pan: {
+    type: double_array,
+    default_value: [-2.35, 2.35],
+    description: "Max values for the head position (in radians)",
+    validation: {
+          fixed_size<>: 2
+        }
+  }
+  max_tilt: {
+    type: double_array,
+    default_value: [-1.2, 0.2],
+    description: "Max values for the head position (in radians)",
+    validation: {
+          fixed_size<>: 2
+        }
+  }
+
+  look_at:
+      tilt_speed: {
+        type: double,
+        default_value: 6.0,
```

Comment:
Maybe add an acceptable range

Reply:
I think when executing look_at, it already check whether the point to look at is in the max_pan and max_tilt and otherwise clips it. Or do you mean the tilt_speed?

Reply:
No you can specify a range in which the parameter is valid. Otherwise it gets rejected by the parameter library.

Reply:
I meant a maximum value for the tilit speed so nobody accidentally bumps it too high.

Reply:
But then i would do the checking twice?